.travis.yml

```diff
@@ -2,7 +2,7 @@ language: go
 
 go:
   - 1.6
-  - tip
+  - 1.7
```

Comment:
I feel like Jordan's approach is probably better so that people who care can look at the errors on `tip` and file upstream bugs as necessary, helps us all be part of a good open source community and all that jazz